packages/spark-core/settings/_colors.scss

```diff
@@ -4,33 +4,26 @@
 
 // Primary
 $red: rgb(200, 16, 46);
-$black: rgb(0, 0, 0);
+$black: rgb(51, 51, 51);
```

Comment:
I saw color changes in another PR you did for Links. I'm guessing you needed it in both PRs?